docs/content/tutorials/gitops-flux/index.md

```diff
@@ -0,0 +1,198 @@
+   ```
+
+   {{< alert title="💡 Required Flux Components" color="info" >}}
+   The Flux source controller is required for this Radius + Flux integration. The `--components=source-controller` flag ensures that the source controller is installed in your cluster. This component is responsible for managing the Git repository and syncing it with your Kubernetes cluster.
+   {{< /alert >}}
+
+   {{< alert title="⚠️ Network Policy" color="warning" >}}
+   The `--network-policy=false` flag is specified here to disable the network policy for the Flux source controller, since by default access to Flux components is restricted. For a production setup, you can instead remove this flag and configure the network policy to allow `radius-system` namespace access to the source controller. This will ensure that the source controller can communicate with your Git repository and sync changes effectively.
+   For more information on the network policy, see the [Flux documentation](https://fluxcd.io/flux/installation/configuration/optional-components/#network-policies).
+   {{< /alert >}}
+
+## Step 3: Initialize a Radius Environment and Application
+
+1. Initialize a new Radius Environment with [`rad init`]({{< ref rad_initialize >}}):
+
+   ```bash
+   rad init
+   ```
+
+   When asked if you want to create a new application select "Yes". This will create a new file named `app.bicep` in your directory where your application will be defined. It will also create a [`bicepconfig.json`]({{< ref "/guides/tooling/bicepconfig/overview" >}}) file that will contain the necessary setup to use Radius types with Bicep.
+
+## Step 3: Prepare Application for Deployment by Flux
+
+1. Open `app.bicep` and see the scaffolded application created by `rad init`:
+
+   {{% rad file="snippets/1-app.bicep" embed=true %}}
+
+1. Update the `app.bicep` file. We want to link the default environment and specify an application to deploy Radius resources to.
+
+   {{% rad file="snippets/2-app-with-environment.bicep" embed=true %}}
+
+   This Radius application is now ready to be deployed. Instead of using the `rad deploy` CLI command, we will use Flux to detect changes in the Git repository and deploy the application automatically.
+
+   Next, we will create a configuration file to inform Radius about which Bicep files to compile and deploy. This file is called `radius-gitops-config.yaml` and it will be created in the same directory as your `app.bicep` file.
+
+   Create a new file named `radius-gitops-config.yaml` and add the following content:
+
+   {{% rad file="snippets/2-radius-gitops-config.yaml" embed=true %}}
+
+   Finally, commit and push your changes to the Git repository:
+
+   ```bash
+   git add app.bicep radius-gitops-config.yaml
+   git commit -m "Add Radius application and configuration for Flux"
+   git push origin main
+   ```
+
+## Step 4: Configure Flux to Manage Your Application
```

Comment:
these Flux specific steps are fine to include as they are core to the Flux+Radius workflow